libp2p/connmanager.nim

```diff
@@ -67,19 +66,14 @@ type
   PeerEventHandler* =
     proc(peerId: PeerId, event: PeerEvent): Future[void] {.gcsafe, raises: [Defect].}
 
-  MuxerHolder = object
-    muxer: Muxer
-    handle: Future[void]
-
   ConnManager* = ref object of RootObj
     maxConnsPerPeer: int
     inSema*: AsyncSemaphore
     outSema*: AsyncSemaphore
-    conns: Table[PeerId, HashSet[Connection]]
-    muxed: Table[Connection, MuxerHolder]
+    muxed: Table[PeerId, seq[Muxer]]
```

Comment:
Why change the HashSet to a Seq?

Reply:
Additionally, it should guarantee there are no duplications.

Reply:
This seq should stay very small, so I don't worry too much about efficiency
A HashSet can introduce surprising ordering behavior, on the other hand